crates/dwallet-mpc-centralized-party/src/lib.rs

```diff
@@ -42,6 +42,29 @@ type EncryptionOfSecretKeyShareAndPublicKeyShare =
 pub type NoncePublicShareAndEncryptionOfMaskedNonceSharePart =
 <AsyncProtocol as twopc_mpc::presign::Protocol>::NoncePublicShareAndEncryptionOfMaskedNonceSharePart;
 
+/// Contains the public keys of the DWallet.
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, schemars::JsonSchema, Hash)]
+pub struct DWalletPublicKeys {
+    pub centralized_public_share: Vec<u8>,
+    pub decentralized_public_share: Vec<u8>,
+    pub public_key: Vec<u8>,
+}
+pub type DKGDecentralizedOutput =
+    <AsyncProtocol as twopc_mpc::dkg::Protocol>::DecentralizedPartyDKGOutput;
+
+/// Extracts [`DWalletPublicKeys`] from the given [`DKGDecentralizedOutput`].
+// Can't use the TryFrom trait as it leads to conflicting implementations.
+// Must use `anyhow::Result`, because this function is being used also in the centralized party crate.
```

Comment:
```suggestion
// Can't use the TryFrom trait as it leads to conflicting implementations.
// Must use `anyhow::Result`, because this function is being used also
// in the centralized party crate.
```